templates/new_structure/BuildStage.yml

```diff
@@ -0,0 +1,81 @@
+parameters:
+- name: checkovImageName
+  type: string
+  default: 'bridgecrew/checkov'
+- name: scanDirectory
+  type: string
+  default: 'terraform'
+- name: terrascanImageName
+  type: string
+  default: 'accurics/terrascan'
+- name: tfsecImageName
+  type: string
+  default: 'aquasec/tfsec'
+- name: skipCheck
+  type: string
+  default: 'none'
+- name: workingDirectory
+  type: string
+- name: checkovTestResultsFileName
+  type: string
+- name: dependsOn
+  type: string
+- name: environment
+  type: string
+#- name: privatePoolName
```

Comment:
Why did this get removed? I think this is something we're going to need.

Reply:
Fixed in new commits.